README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <img src="src/assets/MonokleLogoLight.svg#gh-light-mode-only" alt="Monokle Logo Light"/>
-  <img src="src/assets/MonokleLogoDark.svg#gh-dark-mode-only" alt="Monokle Logo Dark" />
+  <img src="src/assets/MonokleLogoDark.svg" alt="Monokle Logo Dark" />
```

Comment:
any reason to remove `#gh-dark-mode-only` here? 
previewing the README locally will show two logos, but with those hash parameters, while previewing on GitHub it will show the logo based on light/dark mode